lib/views/go_to_page.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:flutter_typeahead/flutter_typeahead.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+class GoToPage extends StatefulWidget {
+  final Function(List<String>) route;
+  GoToPage({this.route});
+
+  @override
+  _GoToPageState createState() => new _GoToPageState();
+  final originField = TextField(
+    decoration: InputDecoration(
+        contentPadding: EdgeInsets.fromLTRB(15.0, 12.0, 10.0, 12.0),
+        border: OutlineInputBorder(borderRadius: BorderRadius.circular(32.0))),
+  );
+}
+
+class _GoToPageState extends State<GoToPage> {
+  //To be able to save the selected suggestion into the text field
+  final TextEditingController _originTypeAheadController =
+      TextEditingController();
+  final TextEditingController _destinTypeAheadController =
+      TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    List<String> originDestination = [];
+    FocusNode destinationTextField =
+    new FocusNode(); // this focus node will be used for the second text field (destination)
+    return Container(
+      child: Material(
+        child: Container(
+          margin: new EdgeInsets.only(left: 15.0, top: MediaQuery.of(context).padding.top + 5.0, right: 15.0),
+          //color: Color.fromRGBO(147, 0, 44, 1),
```

Comment:
Why is this commented out? Should it be deleted?

Reply:
Removed this line.